src/components/Toast.jsx

```diff
@@ -1,4 +1,5 @@
 import PropTypes from 'prop-types';
+import React from 'react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'React' is defined but never used.